tests/collection_test.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+import pygerduty.v2
+
+###################
+# Version 2 Tests #
+###################
+
+def test_id_to_obj():
+
+	kwarg = {
```

Comment:
I have been having issues with my tabs, sometimes it does tabs = four spaces and other times it does not. I will check on it.